Source/CardView.h

```diff
@@ -25,5 +25,7 @@
 @optional
```

Comment:
We do need to be able to add/remove cards on the fly (per Nokkeltall's design) and I thought it's better to have insert/remove methods for that instead of just assigning a new array of view controllers to the `viewControllers` property. Not to mention that if it isn't a big hassle, then it's better to make it a bit more generic (re: reusability).

We might drop one of the `insert` methods, but then we'd have problems with inserting a card after the last one (or before the first one, depending on which one we would remove).